### PR TITLE
OBPIH-4712 restyle the react list table

### PR DESCRIPTION
**Pull request by drodzewicz**

### CODE REVIEW

File: src/js/components/DataTable/DataTable.scss [hunk 1]
```diff
@@ -0,0 +1,215 @@
+.app-react-table-wrapper {
+  //border-radius: 4px;
+  //overflow: hidden;
+}
```
Comment (kchelstowski) [inline src/js/components/DataTable/DataTable.scss:4]: Is it still needed if it's commented?

File: src/js/components/DataTable/DataTable.scss [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        border: 1px solid var(--gray-200);
+        color: var(--gray-600);
+        height: 28px;
+        min-height: 28px;
+        & > * {
+          height: 28px;
+        }
+        &--is-focused {
+
+        }
```
Comment (kchelstowski) [inline src/js/components/DataTable/DataTable.scss:193]: nothing inside `&--is-focused`, should it stay?

File: src/js/components/purchaseOrder/PurchaseOrderListStyles.scss [hunk 3]
```diff
@@ -80,7 +80,6 @@
       }
     }
   }
-  // TODO: Delete this after table's restyling is ready
   .ReactTable .rt-tfoot .rt-td {
     border: 0 !important;
     font-weight: 600;
```
Comment (kchelstowski) [inline src/js/components/purchaseOrder/PurchaseOrderListStyles.scss:85]: I think you can delete this code, as TODO was written to delete this part as soon as your styling comes unless you use it in some way?

File: src/js/utils/ActionDots.jsx [hunk 4]
```diff
@@ -5,39 +5,72 @@ import { RiMoreLine } from 'react-icons/all';
...
+            {action.onClickActionWithId && (
+              <a
+                key={action.label}
+                href="#"
+                onClick={() => action.onClickActionWithId(id)}
+                className={`d-flex align-items-center gap-8 dropdown-item ${action.variant === 'danger' ? 'font-red-ob' : ''}`}
+              >
+                {action.leftIcon && action.leftIcon}
+                {action.label &&
+                <Translate id={action.label} defaultMessage={action.defaultLabel} />}
```
Comment (kchelstowski) [inline src/js/utils/ActionDots.jsx:57]: nice catch, I haven't realized I hardcoded id of translation as `react.purchaseOrder.viewOrderDetails.label` (translations weren't working for me, so I got defaultMessage always instead)

**Review by kchelstowski — CHANGES_REQUESTED** [review]

**Review by awalkowiak — APPROVED** [review]

**Review by kchelstowski — APPROVED** [review]

**Pull request merged**